### ISSUE TITLE
`style` should be overridden from parents for accessibility

### ISSUE THREAD

**Issue by sehilyi** [gosling-lang/gosling.js#362]
```js
{
   style: { outline: "white" }, // use white outline in all child tracks if it is undefined by the child
   views: [...]
}
```
